### ISSUE TITLE
Snack Bug regarding PropType Validation?

### ISSUE THREAD

**Issue by AliShehzad18** [expo/expo#6408]
I posted a question on the expo forums and a Samaritan who goes by _woodin_ said it might be a bug and suggested I post the issue here. 

The issue I have been experiencing is not seeing a yellow warning box on snack when I pass an incorrect Proptype. 

Following is the link to the original question: https://forums.expo.io/t/proptype-validation-on-snack/30580

**Comment by wodin** [comment]
Here's a link to the snack that demonstrates the problem:

https://snack.expo.io/@alishehzad/no-warning-for-incorrect-proptype

This works fine locally. Just not in a snack.

**Comment by AliShehzad18** [comment]
Thank you for the link to a more relevant repository and the transfering process. 

Unfortunately, it seems like I dont have transfer privileges. Hence, Ive created a new issue in that repository. Below is the link:
https://github.com/expo/snack-web/issues/30

**Comment by wodin** [comment]
Yes, I was thinking someone from the Expo team might transfer it, but since there's already a similar issue they may as well close this one.

**Comment by IjzerenHein** [comment]
PropTypes unfortunately doesn't work. As prop-types is no longer cool, I recommend moving to typings.
I've updated the example to TypeScript and it gives you hints in the editor that the `count` type is incorrect.

https://snack.expo.io/@ijzerenhein/no-warning-for-incorrect-proptype

![image](https://user-images.githubusercontent.com/6184593/96894096-ab302180-148b-11eb-83d8-763dbf956696.png)

Is this a viable solution for you?
